5_evaluate/src/eval_utility_fxns.R

```diff
@@ -71,36 +71,42 @@ match_pred_obs <- function(preds_file, eval_obs) {
       mutate(obs_1d, pred = NA)
     })
     return(interp_1d)
-  }))
+  })) %>%
+    filter(!is.na(pred)) # Filter out rows where there are no matching predictions for observed depths (e.g. observed depth exceeds predicted depth)
```

Comment:
I'm fine with this being silent, but as we've noted in some other repos (Andy's LSTM and the lake-prep), having a ton of obs below the max depth of the lake is a potential flag that there is maybe something wrong with the reported max depth and/or the observations. But agreed that doing that QAQC here doesn't make sense and any flagging of that problem should happen in another repo.